ui-grid/ui-grid.d.ts

```diff
@@ -490,6 +490,10 @@ declare module uiGrid {
          */
         footerHeight?: number;
         /**
+         * returns or sets grid height in pixels
+         */
+        gridHeight?: number;
```

Comment:
Can you link to either the source or the docs for the missing stuff?  Is this the official docs description?  This is a property of type number.  It doesn't return anything